datahub-web-react/src/app/entity/user/UserProfile.tsx

```diff
@@ -1,110 +1,435 @@
+    const groupsDetails = data?.corpUser?.relationships as ExtendedEntityRelationshipsResult;
+    const photoUrl = data?.corpUser?.editableProperties?.pictureLink || undefined;
     const contentLoading =
         Object.keys(ownershipResult).some((type) => {
             return ownershipResult[type].loading;
         }) || loading;
 
-    const ownershipForDetails = useMemo(() => {
-        const filteredOwnershipResult: {
-            [key in EntityType]?: Array<SearchResult>;
-        } = {};
-
-        Object.keys(ownershipResult).forEach((type) => {
-            const entities = ownershipResult[type].data?.search?.searchResults;
-
-            if (entities && entities.length > 0) {
-                filteredOwnershipResult[type] = ownershipResult[type].data?.search?.searchResults;
-            }
-        });
-        return filteredOwnershipResult;
-    }, [ownershipResult]);
-
     if (error || (!loading && !error && !data)) {
         return <Alert type="error" message={error?.message || 'Entity failed to load'} />;
     }
 
-    const groupMemberRelationships = data?.corpUser?.relationships as EntityRelationshipsResult;
-
+    // Routed Tabs Constants
     const getTabs = () => {
         return [
             {
-                name: TabType.Ownership,
-                path: TabType.Ownership.toLocaleLowerCase(),
-                content: <RelatedEntityResults searchResult={ownershipForDetails} />,
+                name: TabType.Assets,
+                path: TabType.Assets.toLocaleLowerCase(),
+                content: <UserAssets />,
+                display: {
+                    enabled: () => true,
+                },
             },
             {
                 name: TabType.Groups,
                 path: TabType.Groups.toLocaleLowerCase(),
                 content: (
                     <UserGroups urn={urn} initialRelationships={groupMemberRelationships} pageSize={GROUP_PAGE_SIZE} />
                 ),
+                display: {
+                    enabled: () => groupsDetails?.relationships.length > 0,
+                },
             },
         ].filter((tab) => ENABLED_TAB_TYPES.includes(tab.name));
     };
+    const defaultTabPath = getTabs() && getTabs()?.length > 0 ? getTabs()[0].path : '';
+    const onTabChange = () => null;
 
-    const getHeader = (user: CorpUser) => {
-        const { editableInfo, info } = user;
-        const displayName = entityRegistry.getDisplayName(EntityType.CorpUser, user);
-        return (
-            <UserHeader
-                profileSrc={editableInfo?.pictureLink}
-                name={displayName}
-                title={info?.title}
-                email={info?.email}
-                skills={editableInfo?.skills}
-                teams={editableInfo?.teams}
-            />
-        );
+    // EditProfile modal Constants
+    const getEditModalData = () => {
+        return {
+            urn: data?.corpUser?.urn || undefined,
```

Comment:
Minor: you can use the "urn" field here so you don't need || undefined.

Reply:
okay